build_datacard.py

```diff
@@ -39,7 +39,8 @@ def change_bin_width():
     if binw is not None:
         # Testing different bin widths
         left = 180.
-        right = 720.
+        #right = 720. #used by Thomas but not sure why
+        right = 650.
```

Comment:
changes like this will conflict with #3

Reply:
Okay, I am happy to rebase on top of #3, I have no more comments on it

Reply:
Fixed after rebase